libs/features/georoutingv2/georoutingv2.js

```diff
@@ -178,7 +178,7 @@ function buildContent(currentPage, locale, geoData, locales) {
   const titleText = geo.length ? geo[0][currentPage.geo] : '';
   const title = createTag('h3', { lang, dir }, locale.title.replace('{{geo}}', titleText));
   const text = createTag('p', { class: 'locale-text', lang, dir }, locale.text);
-  const flagFile = getCodes(locale).length > 1 ? 'globe-grid.png' : `flag-${locale.geo}.svg`;
+  const flagFile = locale.globeGrid?.toLowerCase().trim() === 'yes' ? 'globe-grid.png' : `flag-${locale.geo.replace('_', '-')}.svg`;
```

Comment:
nit: excel boolean values are standardized to be "on" and "off" so we should keep it that way for globeGrid

Reply:
Updated in code, excel and PR description.